client/src/LinearGradient.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+function linearGradient() {
+  return (
+    <div class ="bg-gradient-to-r from-pink-500 via-red-500 to-yellow-500">
```

Comment:
Need to remove space before `=`

```suggestion
    <div class="bg-gradient-to-r from-pink-500 via-red-500 to-yellow-500">
```